source/network/ODServer.cpp

```diff
@@ -606,6 +610,7 @@ bool ODServer::processClientNotifications(ODSocketClient* clientSocket)
             const std::string& nick = clientSocket->getPlayer()->getNick();
             int32_t id = clientSocket->getPlayer()->getId();
             int seatId = seat->getId();
+            seat->setMapSize(gameMap->getMapSizeX(), gameMap->getMapSizeY());
```

Comment:
Ok, then I'd propose to either add an if (== EditorMode) if it is feasible there or simply document it.
Both can be done later. :)